build.gradle.kts

```diff
@@ -98,9 +98,13 @@ dependencies {
 }
 
 stutter {
+    isSparse = (findProperty("stutter.sparce")?.toString()?.toBoolean()) ?: true
     java(8) {
         compatibleRange("4.10")
     }
+    java(11) {
+        compatibleRange("4.10")
+    }
```

Comment:
❓ Why do we need this?

Reply:
I was surprised at first, but to have the configuration in stutter applied it is required to call `gw stutterWriteLocks` instead of just making changes in the configuration (I spent some time trying to explain why sparse mode doesn't work, even it was explicitly enabled in the project configuration...
Nevertheless, it is in fact no longer needed as fallback java8.lock will be taken instead. I will revert it.